runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -846,6 +855,20 @@ else if (client.getVarbitValue(Varbits.COMBAT_ACHIEVEMENT_TIER_MASTER) == 2)
 			}
 		}
 
+		if (config.showArceuusCooldown())
+		{
+			final int magicLevel = client.getRealSkillLevel(Skill.MAGIC);
+			if (message.endsWith(MARK_OF_DARKNESS_MESSAGE))
+			{
+				createGameTimer(MARK_OF_DARKNESS_COOLDOWN, Duration.of(magicLevel - 10, RSTimeUnit.GAME_TICKS));
+			}
+			else if (message.endsWith(MARK_OF_DARKNESS_EXPIRED_MESSAGE))
```

Comment:
Does the expired message block need to be gated behind a config check at all? I don't think so. (Also when, if ever, does it fire when the timers aren't naturally expiring on their own?)